syft/pkg/cataloger/dotnet/test-fixtures/image-net8-compile-target/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0-alpine@sha256:3f93439f47fea888d94e6e228d0d0de841f4122ef46f8bfd04f8bd78cbce7ddb AS build
+ARG RUNTIME=win-x64
+WORKDIR /src
+
+COPY src/helloworld.csproj .
+RUN dotnet restore -r $RUNTIME
+
+COPY src/*.cs .
+
+RUN dotnet publish -c Release --no-restore -o /app
+
+# note: we're not using a runtime here to make testing easier... but this functionally will not work if this was a produciton app
```

Comment:
I don't understand what this comment is trying to tell me.

Reply:
I see. "this functionality will not work if this was a production app" sounds like the _cataloger_ wouldn't work on a production app, which really confused me.

You could say something like, "copy compiled app into empty container to prevent extraneous packages in the test case (the app itself wouldn't work, but the artifacts can still be cataloged)" or something like that?

Reply:
adjusted